src/Microsoft.AspNetCore.Server.Testing/Deployers/NanoDeployer/NanoDeployer.cs

```diff
@@ -0,0 +1,251 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Diagnostics;
+using System.IO;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Logging;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Server.Testing
+{
+    public class NanoDeployer : ApplicationDeployer
```

Comment:
Is there anything about this that is exclusive to Nano, or would this work against any remote Windows machine? `RemoteDeployer`?